packages/mgt/src/graph/graph.photos.ts

```diff
@@ -76,9 +77,12 @@ export async function getPhotoForResource(graph: IGraph, resource: string, scope
       .middlewareOptions(prepScopes(...scopes))
       .get()) as Response;
 
-    if (!response.ok) {
+    if (response.status === 404) {
+      return { eTag: null, photo: null };
```

Comment:
Just curious, why return an object with null values instead of a null for 404?

Reply:
Great question - it's so the object can be cached and to indicate that there is no photo in the graph for that entity.

Reply:
That makes sense. Mind adding a comment?

Reply:
Done